pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/renderer/CompositeValueProvider.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular.renderer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.model.elastic.export.tabular.FilterExportConfig;
+
+/**
+ * value provider for COMPOSITE-type questions.  This supports providing a separate string value for each
+ * child question, and delegating each child question to a different appropriate value provider.
+ */
+public class CompositeValueProvider extends TextValueProvider {
+    public List<String> formatRawValues(List<?> rawValues, FilterExportConfig filterConfig, Map<String, Object> formMap) {
+        if (CollectionUtils.isNotEmpty(filterConfig.getChildConfigs())) {
+            List<String> answerVals = new ArrayList<>();
+            if (rawValues != null && rawValues.size() == 1 && rawValues.get(0) instanceof List) {
```

Comment:
I prefer to have method for this condition with good name rather than having comment, because usually comments become out of date very easily and developers are not motivated to update comments.